releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -2,6 +2,10 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.10 - March, 21 2020
```

Comment:
What is the versioning scheme we use for the VSTS Task? I would generally expect addition of new features to bump up the `Minor` version instead of the `Patch` version.